### PR TITLE
Added issue/pr template.

### PR DESCRIPTION
**Pull request by gyliu513**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/metadata/46)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+/kind bug
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
```
Comment (gaoning777) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Would you mind adding "How to reproduce it (as minimally and precisely as possible):" to the bug report?

**Comment by gaoning777** [comment]
@zhenghuiwang @neuromage This is useful guidelines for issue creation, WDYT?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (gyliu513) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @gaoning777 it was already covered in `**What steps did you take and what happened:**` in Line 9

**Comment by zhenghuiwang** [comment]
/lgtm

I see similar github config in faring and katib already.

**Pull request merged**